src/utils/bugzilla/chartJs/burndownDataFormatter.js

```diff
@@ -41,7 +41,7 @@ const bugsByCreationDate = (bugs, startDate) => {
     const today = new Date();
     const todaysDate = `${today.getUTCFullYear()}-${today.getMonth() + 1}-${today.getDate()}`;
     if (lastDataPoint && lastDataPoint.x !== todaysDate) {
-        accumulatedCount.push({ x: todaysDate, y: count });
+        accumulatedCount.push({ x: new Date(todaysDate), y: count });
```

Comment:
Is this safe?   Javascript Dates have always been an ugly mess of timezones (or the wrong timezone, or utc, or not).  I suggest sticking with milliseconds since epoch for all time variables until they are ready for presentation to the user.

Reply:
This is for presentation.